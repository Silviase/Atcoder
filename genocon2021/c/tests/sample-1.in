4
GGAGGTTATTGACTTCGCCTC
GGTCACTCGATATCTGTGAGCTCGATATCTGTGAGCTCGATATCTGTGAGCC
GGGTTTAACTCGTCAGCCTCGGGTTTAACTCGTCAGCCTC
TGACTTCGCCTCGGTCACTCGATATCTGTGAGCTCGATATCTG