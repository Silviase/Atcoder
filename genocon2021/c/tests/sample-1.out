GGAGGTTA-TTGCT--GTGGAG-GTAC-TGGAGA-AGGA-GGA-TGCTAGCG-TT-GGGT-AAACCAC-G-AGC-ATTTTGACTT-G-T-ACT--TC-GCCTC----
--GGGTTA-TTGCT--GTTGTGAGTAC-TGGAGACAGGAGGGAGTGTTAGAG-TTGGGGT-AAACCACAGTAGCTCATGTCACTTGGATAACTCGTCAGCCTC----
---GGTCACTCGCT--GTGGAGAGTACTTTGAGACAGGAGGGAGTGCTAGAGTTTGGGGTAAAACCACAGCAGCTCATG-CACTTGGATATCT-GTGAG-C-C----

